src/libs/ascent/runtimes/flow_filters/ascent_runtime_rendering_filters.cpp

```diff
@@ -1052,8 +1052,8 @@ DefaultRender::verify_params(const conduit::Node &params,
                              conduit::Node &info)
 {
     info.reset();
-    bool res = check_string("image_name",params, info, false);
-    res &= check_string("image_prefix",params, info, false);
+    bool res = check_path("image_name",params, info, false);
+    res &= check_path("image_prefix",params, info, false);
```

Comment:
I think we should check this on execution, not not on verify params.
We should also only check on rank 0 :-)